### PR TITLE
Unit test run for the TODO-LIST project pull request

### PR DESCRIPTION
**Pull request by AleWaweru**
Installed Jest 
Did a unit test for the TODO-LIST project.

### CODE REVIEW

File: src/modules/todo.test.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  });
+
+  test('when the the user deletes task the task should not be avialibale', () => {
+    expect(deleteTask({
+      task: 'Delete Task 2',
+      id: 1,
+      status: false,
+    })).toStrictEqual([]);
+  });
+});
```
Comment (julie-ify) [inline src/modules/todo.test.js:41]: - [ ] You have done a good job adding tests for your pure functions but you need to mock the local storage, or DOM and add tests for them also. Kindly do that.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @AleWaweru and @wolterbwambale,

Good job so far!

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahmetbozaci** [comment]
### Hello @AleWaweru @wolterbwambale 
### _This is an additional review as a result of the second opinion request._ 

- I noticed that you used Jest version 29, which automatically handles mocking the local storage. There's no need to manually mock it.
- [ ] Please resubmit this pull request, as the next code reviewer will likely approve it 


___


### Cheers and Happy coding! 💻 👏
____
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by AdnanOlarmmi — APPROVED** [review]
Hi @AleWaweru,

This is Adnan Is-haq, your beloved code reviewer, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved office man

## Highlights
- Tests are passing.

## Optional suggestions
_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**